MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+recursive-include geospaas/viewer/templates *
+recursive-include geospaas/viewer/static *
```

Comment:
Maybe we can use these patterns?
```
recursive-include **/templates *
recursive-include **/static *
```
That way we include template and static files for all apps, not just `viewer`

Reply:
Done